packages/sync-actions/test/product-sync-prices.spec.js

```diff
@@ -340,6 +480,95 @@ describe('Actions', () => {
       )
     })
 
+    test('should build `changePrice` action without deleting `custom` prop', () => {
```

Comment:
this test is to validate that `custom` wont be changed or removed if `changePrice` action produced.